.travis.yml

```diff
@@ -1,16 +1,17 @@
-language: node_js
-node_js:
-- '4'
+sudo: true
+
+language: cpp
+
+services:
+  - docker
+
+before_install:
+  - wget https://raw.githubusercontent.com/tue-robotics/tue-env/master/ci/install-package.sh
+  - wget https://raw.githubusercontent.com/tue-robotics/tue-env/master/ci/build-package.sh
+  - export PACKAGE=${TRAVIS_REPO_SLUG#*/}
+
 install:
-- npm install
-script: npm run build
-deploy:
-  provider: releases
-  api_key:
-    secure: W54Oa7xps+yMVd8wUQ/GCrgemcipZ8WTFL/lPGCZeLsqzA6hZ2CzevH/CigrZTmO0eQFIUfZKALJR7EzR9IbsSbIX4b6rgdpRZW6TX2bM0zIncX0JPyXxgnM2WIA5MiL2o7c00U83mqFyOd/BHEeSX9CP0TTL7rEa9HfWNxpo2U=
```

Comment:
@Rayman when we put this back, making this a secret environment variable can't hurt, right?